### PR TITLE
Support new types of errors in charts and selectors

### PR DESCRIPTION
**Pull request by mournfulCoroner**

### CODE REVIEW

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9982-support-new-types-of-errors-1516/common-errors-1516/ru/">common.errors (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9982-support-new-types-of-errors-1516/common-errors-1516/en/">common.errors (en)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
Please merge the Pull Request with the changes from Weblate into your branch.

File: src/ui/libs/DatalensChartkit/Error/Error.tsx [hunk 1]
```diff
@@ -237,6 +237,12 @@ const ChartKitError: React.FC<any> = (props) => {
                 ) {
                     errorDetails.push(i18n('component.chartkit-error.codes', errorCode));
                     detailsString = source.debug.db_message;
+                }
+                // TODO: Replace with specific code
```
Comment (flops) [inline src/ui/libs/DatalensChartkit/Error/Error.tsx:241]: Why we didn't make the specific code in this PR?

**Review by flops — CHANGES_REQUESTED** [review]

File: src/ui/components/DashKit/plugins/GroupControl/utils.ts [hunk 2]
```diff
@@ -58,3 +66,20 @@ export const filterSignificantParams = ({
 
     return dependentSelectors || !defaults ? params : pick(params, Object.keys(defaults));
 };
+
+export const getErrorTitle = (errorInfo: SelectorError) => {
+    const datasetsStatus =
+        errorInfo?.details && 'sources' in errorInfo.details && errorInfo.details.sources
+            ? Object.keys(errorInfo.details.sources)[0]
+            : '';
```
[the remark this answers is not in the document]
Comment (mournfulCoroner) [reply src/ui/components/DashKit/plugins/GroupControl/utils.ts]: replaced with null

File: src/ui/libs/DatalensChartkit/Error/Error.tsx (hunk 1 above)
Comment (mournfulCoroner) [reply src/ui/libs/DatalensChartkit/Error/Error.tsx:241]: waiting for backend

**Review by flops — APPROVED** [review]

**Pull request merged**